### DISCUSSION TITLE
Slack connection not showing up in MWAA UI

### DISCUSSION THREAD

**Discussion by alibinshahid92** [apache/airflow#30143] [Q&A]
Currently, I've got an MWAA env setup. I am trying to create a slack connector via UI but for some reason it's not showing up.

I've tested it locally it is working just fine.

My requirements.txt file contains

```
--constraint "s3://<full address>/constraints-3.7-mod.txt"

    apache-airflow>=2.3.0
    python>=3.10
    apache-airflow-providers-amazon>=7.2.0
    jinja2>=3.1.2
    apache-airflow-providers-slack>=7.2.0
    apache-airflow-providers-http>=4.2.0
    apache-airflow-providers-common-sql>=1.3.1
    apache-airflow-providers-snowflake==2.3.0
    slack_sdk>=3.0.0
```

I've taken the constraints file from [here](https://docs.aws.amazon.com/mwaa/latest/userguide/airflow-versions.html) and added it to my DAG folder.

Any idea where am I going wrong..?

**Comment by potiuk** [comment]
You should likely raise the questio to the MWAA support, not here.
